cdc/entry/schema_storage.go

```diff
@@ -78,9 +79,9 @@ func NewSchemaStorage(
 	)
 	if meta == nil {
 		snap = schema.NewEmptySnapshot(forceReplicate)
-		snap.InitPreExistingTables()
+		// snap.InitPreExistingTables()
```

Comment:
Why is this init phase commented?

Reply:
I forgot to remove this line. Actually, `InitPreExistingTables()` is removed in this PR.